pages.nl/common/git-log.md

```diff
@@ -0,0 +1,36 @@
+# git log
+
+> Geef de commit geschiedenis.
+> Meer informatie: <https://git-scm.com/docs/git-log>.
+
+- Geef de commit reeks startend bij de huidge, in omgekeerde volgerde:
+
+`git log`
+
+- Geef de geschiedenis van een specifieke bestand of map, inclusief de veranderingen:
+
+`git log -p {{pad/naar/bestand_of_map}}`
+
+- Geef een overzicht van welk(e) bestand(en) zijn veranderd in elke commit:
+
+`git log --stat`
+
+- Geef een grafiek van commits in de huidige branch:
+
+`git log --graph`
+
+- Geef de commit reeks, met enkel de eerste lijn van de commit weergegeven:
+
+`git log --oneline`
```

Comment:
Why did you split this example in two and why did you remove the `--oneline --decorate --all --graph` example?

Reply:
I thought it'd clarify that those are two separate options, sorry 😬

Reply:
No problem. If you have suggestions for the page itself, please make a Pull Request to change the original page. All translations should be in sync with the English pages.